src/main/java/se/sundsvall/billingdatacollector/integration/opene/kundfakturaformular/mapper/MapperHelper.java

```diff
@@ -0,0 +1,203 @@
+package se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.mapper;
+
+import static java.util.Optional.ofNullable;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.zalando.problem.Status.INTERNAL_SERVER_ERROR;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+import org.zalando.problem.Problem;
+
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.OpeneCollections;
+import se.sundsvall.billingdatacollector.integration.opene.kundfakturaformular.model.OrganizationInformation;
+
+@Component
```

Comment:
Does this class have to be a `@Component`

Reply:
No